tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -1563,9 +1563,10 @@ public void TestFindWithinRectangle()
             // note: the hits are unordered
         }
 
-        [Fact]
+        [SkippableFact]
         public void TestFindWithMaxScan()
         {
+            RequireServer.Check().VersionLessThan("4.0.0");
```

Comment:
Should be VersionLessThanOrEqual("4.0.0") as maxScan is being removed in 4.2

Reply:
I changed it to VersionLessThan("4.1.0-").

That skips the test on all 4.1.x servers including pre-release versions.